gentrap/src/main/scala/nl/lumc/sasc/biopet/pipelines/gentrap/Gentrap.scala

```diff
@@ -136,6 +136,20 @@ class Gentrap(val parent: Configurable)
     )
   )
 
+  override def biopetScript(): Unit = {
+    val validate = new ValidateAnnotation(this)
+    validate.refflatFile = annotationRefFlat.get
+    validate.gtfFile = fragmentsPerGene.map(_.annotationGtf)
```

Comment:
Nope, gtfFile is a optional argument. The tool also works with only a refflat file.
Cufflinks your right about that, will see if I can put this somewhere in.